build_forklift_bazel.sh

```diff
@@ -4,12 +4,16 @@ echo "Running $0"
 # Change the dir to FORKLIFT_DIR (default forklift)
 cd ${FORKLIFT_DIR:-forklift}
 
+# Copy the stub-images under the bazel workspace
+cp -fr ../stub-images .
```

Comment:
see: we copy the stub-images inside the forklift/ after its cloned

Reply:
This assumes that `FORKLIFT_DIR` is not set, right?

Reply:
> This assumes that `FORKLIFT_DIR` is not set, right?

as long we are at the forklift repo directory , it should work regardless the name.

Reply:
Well since it's not documented anywhere I may be missing a context here. Does some other tool set this or is it configured bu user? Can `FORKLIFT_DIR` name only a subdirectory which is a direct descendant (e.g. `FORKLIFT_DIR="my-pretty-forklift"`) or can it be a completely different path (e.g. `FORKLIFT_DIR=/tmp/forklift`)?

Reply:
well its set by our CI scripts .
we can assume its a forklift because its being cloned here:
https://github.com/kubev2v/forkliftci/blob/main/get_forklift_bazel.sh#L4

so this the structure after the clone:
<forkliftci>
    <stub-images>
    <cloned_forklift>
          WORKSPACE
          
so  assuming we are inside the <clone_forklift> directory , we can copy stub-images from the parent and be under the same workspace.

Reply:
Yes, but `get_forklift_bazel.sh` does not set `FORKLIFT_DIR`. So what does? Either we don't need it and #3 can be reverted, or there is some use-case for it and your patch needs to be adapted to avoid breaking the use-case.

Reply:
well if FORKLIFT_DIR not set that its get its default 'forklift' value (which is most likely to happen)
https://wiki.bash-hackers.org/syntax/pe#use_a_default_value

Reply:
That is not the point. However (un)likely it may be, it still can be set arbitrarily and it seems to be a valid use-case. If such use is invalid (and we require the directory to be a direct descendant) the variable should be validated to prevent any errors later in the code and ideally it's use should also be document.

Reply:
added verification that we are on the correct location and `set +e` which will bail out  if something fails.

Reply:
One note about the paths. 
When the forklift GitHub action starts the forkliftci. The forkliftci does not have forklift directory inside it but the forklift should have the forkliftci (I'm not sure if it is direct or if the GitHub actions clone the repo to different path). That's why we need to have the variable and bit worried that here it might fail.